meta-xt-prod-devel-rcar-control/recipes-guests/domd/files/domd-set-root

```diff
@@ -36,6 +38,14 @@ if [ $BOOT_STORAGE = nfs ] ; then
 
     echo "Mangling domain configuration: setting storage to network boot"
     sed -i "s/STORAGE_PART[0-9]/${BOOT_STR}/g" $DOMD_CFG_FILE
+
+    if [ "$MODE" == "enable_virtio" ]; then
```

Comment:
Why are you doing this in the runtime? We already know configuration during build time.

Reply:
yes, this can and will be simplified